src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpSettings.cs

```diff
@@ -276,29 +248,15 @@ private string DetermineEntityPath(string connectionString = null)
             return EntityName;
         }
 
-        private ServiceBusProcessor CreateProcessor(ServiceBusClient client, string entityName, string subscriptionName)
-        {
-            ServiceBusProcessorOptions options = DetermineMessageProcessorOptions();
-
-            if (string.IsNullOrWhiteSpace(subscriptionName))
-            {
-                return client.CreateProcessor(entityName, options);
-            }
-
-            return client.CreateProcessor(entityName, subscriptionName, options);
-        }
-
-        private ServiceBusProcessorOptions DetermineMessageProcessorOptions()
+        private ServiceBusReceiverOptions DetermineMessageReceiverOptions()
         {
-            var messageHandlerOptions = new ServiceBusProcessorOptions();
+            var options = new ServiceBusReceiverOptions();
             if (Options != null)
             {
-                messageHandlerOptions.AutoCompleteMessages = Options.AutoComplete;
-                messageHandlerOptions.MaxConcurrentCalls = Options.MaxConcurrentCalls;
-                messageHandlerOptions.PrefetchCount = Options.PrefetchCount;
+                options.PrefetchCount = Options.PrefetchCount;
             }
```

Comment:
Just wondering what we now do with MaxConcurrentCalls ?  This is quite an important setting but I don't immediately see from this PR where we are now using this ?  
This controls how many messages we should retrieve from the bus in one batch.

Reply:
Ok, I've found it here in the `StartProcessingMessagesAsync` method:

```
IReadOnlyList<ServiceBusReceivedMessage> messages = 
                        await _messageReceiver.ReceiveMessagesAsync(Settings.Options.MaxConcurrentCalls, cancellationToken: _receiveMessagesCancellation.Token);
```

Reply:
Yes, indeed.